py7zr/py7zr.py

```diff
@@ -939,6 +908,27 @@ def reset(self) -> None:
             self._reset_worker()
             self._reset_decompressor()
 
+    def test(self) -> bool:
+        self._reset_worker()
+        crcs = self.header.main_streams.packinfo.crcs
+        if crcs is not None and len(crcs) > 0:
```

Comment:
When opened archive do not have a CRC32 of packed chunks, it returns True even any check has never ran.
it can be three state, True -- Passed the crc check, False -- fails the check, and None -- no check ran.